lib/ice_cube/time_util.rb

```diff
@@ -109,10 +109,10 @@ def self.deserialize_time(time_or_hash)
       when Time, Date
         time_or_hash
       when DateTime
-        Time.local(time.year, time.month, time.day, time.hour, time.min, time.sec)
+        Time.local(time_or_hash.year, time_or_hash.month, time_or_hash.day, time_or_hash.hour, time_or_hash.min, time_or_hash.sec)
```

Comment:
So I *think* that this was a bug. Not sure when, if ever, the `when DateTime` condition is triggered, but before you had `Time.local(time.year ... ` but the variable `time` doesn't appear to be defined anywhere.

I *think* this was a copy-paste error from the `serialize_time()` method above.